plugins/woocommerce/includes/wc-product-functions.php

```diff
@@ -392,7 +392,7 @@ function wc_get_formatted_variation( $variation, $flat = false, $include_names =
 			// If this is a term slug, get the term's nice name.
 			if ( taxonomy_exists( $name ) ) {
 				$term = get_term_by( 'slug', $value, $name );
-				if ( ! is_wp_error( $term ) && ! empty( $term->name ) ) {
+				if ( ! is_wp_error( $term ) && "" !== $term->name ) {
```

Comment:
Thanks, this looks good. WordPress generally prefers using single quotes when the string doesn't need to be interpolated ([ref](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#single-and-double-quotes)), can you please update? Then it should be good to go :)